package.json

```diff
@@ -88,7 +89,7 @@
     "ember-cli-update": "1.0.1",
     "ember-component-css": "0.8.1",
     "ember-composable-helpers": "5.0.0",
-    "ember-concurrency": "2.3.4",
+    "ember-concurrency": "^2.3.7",
```

Comment:
Why the update?

Reply:
@derrabauke it's updated when i install `Sentry` package